RegistrationApp/RegistrationApp/ViewModels/TaskListViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using RegistrationApp.Models;
+
+namespace RegistrationApp.ViewModels
+{
+    public class TaskListViewModel : BaseViewModel
+    {
+        private List<string> _taskList;
+        public List<string> TaskList
+        {
+            get { return _taskList; }
+            set { SetField(ref _taskList, value); }
+        }
+
+        User user;
+
+        public TaskListViewModel()
+        {
+            user = new User()
```

Comment:
Why do you create new 'User' if you know his pass and username when you redirected to TaskListPage